core/src/main/java/com/scalar/db/common/ActiveTransactionManagedDistributedTransactionManager.java

```diff
@@ -32,6 +34,17 @@ public abstract class ActiveTransactionManagedDistributedTransactionManager
 
   private final ActiveExpiringMap<String, ActiveTransaction> activeTransactions;
 
+  private final AtomicReference<BiConsumer<String, DistributedTransaction>>
```

Comment:
Just a question.
I think we also have this expiration handler in ScalarDB Cluster, but I'm wondering if we need to have it in both sides: ScalarDB core and ScalarDB Cluster.
If the core implements it, the one in ScalarDB Cluster is not necessary?

Reply:
Sorry, I can't remember this expiration handler in ScalarDB Cluster. Please give me some more information about that?